pyEpiabm/pyEpiabm/toy_population_config.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+from .population import Population
+from .household import Household
+
+
+class ToyPopulation:
+    """ Class that creates a toy population for use in the simple python model.
+    """
+    def __init__(self, pop_size: int, cell_number: int,
+                 microcell_per_cell: int, household_number: int,
```

Comment:
C++ currently using n_cells, n_microcells, n_households. Python version doesn't have to match this, but probably worth keeping the python consistent within itself (so either use X_per_Y or X_number instead of both)